topiary-cli/tests/sample-tester.rs

```diff
@@ -5,137 +5,169 @@ use topiary_core::test_utils::pretty_assert_eq;
 
 use tempfile::TempDir;
 
-fn io_test(file: &str) {
-    let input = PathBuf::from(format!("tests/samples/input/{file}"));
-    let expected = PathBuf::from(format!("tests/samples/expected/{file}"));
-
-    // Make sure our test makes sense
-    assert!(input.exists() && expected.exists());
-
-    // Load the known good formatted file
-    let expected_output = fs::read_to_string(&expected).unwrap();
-
-    // Stage the input to a temporary directory
-    let tmp = TempDir::new().unwrap();
-    let staged = tmp.path().join(file);
-    fs::copy(input, &staged).unwrap();
-
-    // Run Topiary against the staged input file
-    let mut topiary = Command::cargo_bin("topiary").unwrap();
-    let output = topiary
-        .env("TOPIARY_LANGUAGE_DIR", "../topiary-queries/queries/")
-        .arg("fmt")
-        .arg("-vvv")
-        .arg(&staged)
-        .output()
-        .expect("Failed to run `topiary fmt`");
-
-    // Print the invocation output, so that it can be inspected with --nocapture
-    let output_str = String::from_utf8(output.stderr).expect("Failed to decode Topiary output");
-    println!("{output_str}");
-
-    // Read the file after formatting
-    let formatted = fs::read_to_string(&staged).unwrap();
-
-    // Assert the formatted file is as expected
-    pretty_assert_eq(&expected_output, &formatted);
-}
-
-fn coverage_test(file: &str) {
-    let input = PathBuf::from(format!("tests/samples/input/{file}"));
-
-    // Make sure our test makes sense
-    assert!(input.exists());
-
-    // Run `topiary coverage` against the input file
-    let mut topiary = Command::cargo_bin("topiary").unwrap();
-    let output = topiary
-        .env("TOPIARY_LANGUAGE_DIR", "../topiary-queries/queries/")
-        .arg("coverage")
-        .arg(&input)
-        .output()
-        .expect("Failed to run `topiary coverage`");
-
-    if !output.status.success() {
-        let output_str = String::from_utf8(output.stdout).expect("Failed to decode topiary output");
-        panic!("Insufficient coverage of \"{file}\":\n{output_str}")
+fn get_file_stem(language: &str) -> &str {
```

Comment:
`extension`, rather than `stem` (stem is usually the bit before the extension).

Reply:
whoops, switched them up!